src/mon/Monitor.cc

```diff
@@ -3697,6 +3700,59 @@ void Monitor::handle_command(MonOpRequestRef op)
     rdata.append(ds);
     rs = "";
     r = 0;
+  } else if (prefix == "mon ok-to-stop") {
```

Comment:
i feel that "mon ok-to-stop" is a superset of "mon ok-to-rm" functionality-wise. as "mon ok-to-stop" allows user to specify multiple monitors by their id, while the latter only allow a single monitor id. the difference is that, the latter complains if the id is not in monmap, while the former just ignores the ones not in monmap. both of them returns "0" in this case. and the test does test the behavior with
```sh
ceph mon ok-to-stop dne || return 1
ceph mon ok-to-rm dne || return 1
```
why not returning `-EINVAL` if an invalid monitor id is specified?

and why do we need two commands for the same feature? one for monitors (in the plural) and one for monitor (in the singular)? and IMHO, the different notations of "stop" and "rm" are confusing to user.

Reply:
the rm command checks if you can remove the mon from the monmap, while the stop one checks whether you can stop the daemon.

Reply:
ahh, i see the difference now. the "rm" command will actually *modify* the monmap. so there are chances that "rm" a monitor won't break the quorum, while "stop" will.